lib/Event/DocumentEvents.php

```diff
@@ -115,63 +115,6 @@ final class DocumentEvents
      */
     const POST_LOAD = 'pimcore.document.postLoad';
 
-    /**
-     * Processor contains the processor object used to generate the PDF
-     *
-     * Arguments:
-     *  - processor | instance of the PDF processor Pimcore\Web2Print\Processor\{ProcessorName}
-     *
-     * @Event("Pimcore\Event\Model\DocumentEvent")
-     *
-     * @var string
-     */
-    const PRINT_PRE_PDF_GENERATION = 'pimcore.document.print.prePdfGeneration';
```

Comment:
might need deprecation and alias in 10.6

Reply:
We should also add upgrade notes to 11.x that these events has been moved.

Reply:
https://github.com/pimcore/pimcore/issues/14418 created an issue for this.